src/routes/spellen/[game].svelte

```diff
@@ -35,28 +35,31 @@
 	import type { Game, Material } from '$lib/games';
 
 	/* Components */
+	import GameReview from '$lib/GameViews/GameReview.svelte';
 	import MaterialButton from '$lib/shared/Button/MaterialButton.svelte';
 	import Accordion from '$lib/shared/Modals/Accordion.svelte';
 	import MaterialModal from '$lib/shared/Modals/MaterialModal.svelte';
 	import CardLabel from '$lib/shared/Label/CardLabel.svelte';
 	import OfflineLabel from '$lib/shared/Label/OfflineLabel.svelte';
 	import DownloadButton from '$lib/shared/Button/DownloadButton.svelte';
+	import Snackbar from '$lib/shared/Snackbar/Snackbar.svelte';
```

Comment:
```suggestion
```

Reply:
Zag het ja, maar das niet mijn ding. @RooyyDoe is hier toch de snackbar, of wah.